### PR TITLE
Update repo and image tag to use gcr

### PR DESCRIPTION
**Pull request by lyuboxa**
### Description

Use released tags

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/conduitio/conduit-operator/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

**Review by lovromazgon — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 SHELL = /usr/bin/env bash -o pipefail
 
-REPO              ?= conduitio/conduit-operator
+REPO              ?= ghcr.io/conduitio/conduit-operator
```
Comment (hariso) [inline Makefile:3]: I'm wondering if it would be enough to replace this with `conduit.docker.scarf.sh/conduitio/conduit`.

Comment (hariso) [reply Makefile:3]: Sorry, bad place.:D Yes, that.

Comment (lyuboxa) [reply Makefile:3]: PRs welcome ;-)

Comment (hariso) [reply Makefile:3]: Will do, was just double-checking with you.:)

**Pull request merged**